### DISCUSSION TITLE
How could I customize my own logging information?

### DISCUSSION THREAD

**Discussion by liming-ai** [open-mmlab/mmdetection#9115] [Q&A]
### What is the problem this feature will solve?

The existing logger is only available for pre-defined scalars, such as loss, lr, etc.

How could we log other customized information?

I cannot find any tutorials about it, thought I think it is very important for users.

### What is the feature you are proposing to solve the problem?

The existing logger is only available for pre-defined scalars, such as loss, lr, etc.

How could we log other customized information?

I cannot find any tutorials about it, thought I think it is very important for users.

**Comment by ZwwWayne** [comment]
The logging information is defined by LoggerHook https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/text.py. In MMDet 2.x, is is inconvenient to log something you want. We have enhance that in MMDet 3.x, where you can find related tutorials in MMEngine [here](https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/logging.html)
